### ISSUE TITLE
Migrate settings to match structure of My Calendar settings

### ISSUE THREAD

**Issue by joedolson** [joedolson/my-calendar-pro#48]
Easier management.

**Comment by paaljoachim** [comment]
Ahh you mean changing this:
![Screenshot 2023-03-09 at 00 13 39](https://user-images.githubusercontent.com/5323259/223873940-b33348b6-d8c1-4208-b048-5df5b74a9e21.jpg)


To become similar to this:
![Screenshot 2023-03-09 at 00 13 30](https://user-images.githubusercontent.com/5323259/223873880-967fa5a9-b523-4b3f-92fb-b7f6afe7076c.jpg)

**Comment by joedolson** [comment]
No; I mean changing the internal data structure so that everything is accessed through a single array of My Calendar Pro options rather than individual database options. This is data structures only, and has nothing to do with the UI.

**Comment by paaljoachim** [comment]
Alrighty!
